steps/id-and-residency/tenant-photo.tsx

```diff
@@ -1,3 +1,4 @@
+import yesNoNotPresentRadio from "helpers/yesNoNotPresentRadio";
```

Comment:
```suggestion
import yesNoNotPresentRadio from "../../helpers/yesNoNotPresentRadio";
```

Reply:
This still seems to br failing circle ci even after rebase.

Reply:
Suggest you review your code and see where the issue is.

Reply:
Also, your rebase seems to have reverted the changes to add the not present button to the page.